### ISSUE TITLE
🐛 [Bug] - The Swedish translation is unusable

### ISSUE THREAD

**Issue by dbratell** [frappe/books#850]
### Expected Behavior

I tried frappe in Swedish, curious what it would look like. I didn't have many expectations but I expected to see Swedish. I also expected to see terms I'm familiar with.

### Current Behavior

The Swedish translation is full of words that do not match the meaning. If it had only been a few cases I could have listed it, but it's basically everything.

Just some examples: "nolloros" is not a word at all, found in the dialogue to create a number series (assuming that word is translated correcty). There are fields for some kind of currency input that are labeled "betyg" as in "grade in a school subject". "Akkurat" might be Norweigan or Danish, but there is no double k in Swedish and even replacing it with a ck, it's still not a word. "Allmänning" which I assume comes from something "general" refers to a shared grassy area in a village. Under "Allmänning" there is an entry "Fest", which means "Party"..

Basically it's all wrong. I think the number of correctly translated words and phrases are in a small minority.

### Steps to Reproduce

1. Start the program.
2. Select Swedish as language.
3. Maybe restart it.

### FrappeBooks Version

0.21.1

### Path or Feature name

Translations

### Country

SE

### Language

SV

### OS

Windows 10

**Comment by Isaac-GC** [comment]
Hi @dbratell!

Translations are still very much a WIP! If there are better translated words/phrases, would you be willing to update or add the translations in this file please? https://github.com/frappe/books/blob/master/translations/sv.csv
